rewrite-maven/src/main/java/org/openrewrite/maven/internal/MavenPomDownloader.java

```diff
@@ -665,6 +666,11 @@ public MavenRepository normalizeRepository(MavenRepository originalRepository, @
             if (repository.getUri().contains("${")) {
                 return null;
             }
+            // Skip blocked repositories
+            // See https://github.com/openrewrite/rewrite/issues/3141
+            else if (repository.getUri().contains("0.0.0.0")) {
+                return null;
+            }
```

Comment:
Trying to resolve these is a waste of time so it should be safe to skip them. Unless you think that we should just use the retry configuration?

Reply:
I think it's fine to short circuit here indeed; might be a slightly specific, but clearer than a retry failure.